src/lib.rs

```diff
@@ -134,13 +134,48 @@ impl fmt::Display for Cesu8DecodingError {
 ///            from_cesu8(data).unwrap());
 /// ```
 pub fn from_cesu8(bytes: &[u8]) -> Result<Cow<str>, Cesu8DecodingError> {
+    from_cesu8_internal(bytes, /*is_java*/false)
```

Comment:
Rather than write `/*is_java*/`, you could use:

```
enum Variant {
    Standard,
    Java,
}
```

Which would make this `from_cesu8_internal(bytes, Variant::Standard)`. This is self-documenting, easier to extend (if somebody invents a third variant of this horrible format), and compiles to the same code.

I'll change this locally after merging; no need to resubmit.